### DISCUSSION TITLE
recipes/cookies-and-bars/persian-chickpea-cookies-with-pistachio/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#996] [Comments]
# recipes/cookies-and-bars/persian-chickpea-cookies-with-pistachio/

A collection of my recipes.

https://nicholaswilde.io/recipes/cookies-and-bars/persian-chickpea-cookies-with-pistachio/

**Comment by nicholaswilde** [comment]
Add oil to mixture until dough comes together
